ext/sockets/sockets.c

```diff
@@ -1248,8 +1248,8 @@ PHP_FUNCTION(socket_strerror)
 		Z_PARAM_LONG(arg1)
 	ZEND_PARSE_PARAMETERS_END();
 
-	if (ZEND_LONG_EXCEEDS_INT(arg1)) {
-		zend_argument_value_error(1, "must be between %d and %d", INT_MIN, INT_MAX);
+	if (arg1 <= INT_MIN || arg1 > INT_MAX) {
+		zend_argument_value_error(1, "must be greater than %d and lower than %d", INT_MIN, INT_MAX);
```

Comment:
The standard wording is:
```suggestion
		zend_argument_value_error(1, "must be between %d and %d", INT_MIN, INT_MAX);
```

Reply:
Yeah, but in this case `INT_MIN` is no longer supported, so should be:

```suggestion
		zend_argument_value_error(1, "must be between %d and %d", INT_MIN + 1, INT_MAX);
```

But I'm still not convinced that we need to constrain the allowed range.